src/airport.js

```diff
@@ -8,19 +8,10 @@ class Airport {
 
 
   landPlane(plane) {
-    if (this.checkPlaneValid(plane) === undefined) {
-      if (this.checkPlaneFlying(plane) === true) {
-        if (this.checkCapacity() === false) {
-          plane.land();
-        } else {
-          return `Cannot land plane, Airport is at capacity`;
-        }
-
-      } else {
-        return `This plane cannot be landed`;
-      }
+    if (this.checkPlaneValid(plane) === undefined && this.checkPlaneFlying(plane) === true && this.checkCapacity() === false) {
```

Comment:
I knew that the nested if statements were not the most clean way of writing the code. Here I refactored it so that all of the conditions to land the plane were checked in one line. Although I still question if this is the best way I could have done this.